.gen/proto/history/v1/service.pb.go

```diff
@@ -33,9 +33,9 @@ import (
 
 	proto "github.com/gogo/protobuf/proto"
 	types "github.com/gogo/protobuf/types"
-
```

Comment:
I think we should ignore generated files. Maybe we can skip .gen?

Reply:
we haven't historically, and there isn't really any downside (it takes about 1 second to reformat everything).

tbh in most ways it's easier to format everything than to exclude stuff.

Reply:
Makes sense

Reply:
ah, one exception to all this is when the generate-and-format makefile isn't being careful about dependencies.  then having two things touch the same files can be a huuuge pain, and tbh I suspect that's why a lot of things avoid formatting gen.

but that's fixed!  our makefile should be actually-completely reliable about this.